### PR TITLE
Add GH to Jira automation

### PR DESCRIPTION
**Pull request by Deezzir**
Adds the bot to the dcgm-snap repo so that new bugs can automatically appear in jira.

### CODE REVIEW

File: .github/.jira_sync_config.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  # Dictionary mapping GitHub issue status to Jira issue status
+  status_mapping:
+    opened: Untriaged
+    closed: done
+
+  # (Optional) Jira project components that should be attached to the created issue
+  # Component names are case-sensitive
+  components:
+    - hardware-observer
```
Comment (gabrielcocenza) [inline .github/.jira_sync_config.yaml:18]: I think this should be changed to dcgm-snap. Can you confirm @aieri

Comment (Deezzir) [reply .github/.jira_sync_config.yaml:18]: There is no such component; I can create one, but thought we could use `hardware-observer` as it already exists

Comment (aieri) [reply .github/.jira_sync_config.yaml:18]: Components are free, so I have created the `dcgm` one.

**Review by aieri — CHANGES_REQUESTED** [review]

File: .github/.jira_sync_config.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    opened: Untriaged
+    closed: done
+
+  # (Optional) Jira project components that should be attached to the created issue
+  # Component names are case-sensitive
+  components:
+    - dcgm
+
+  # (Optional) (Default: None) Parent Epic key to link the issue to
+  epic_key: SOLENG-46
```
Comment (chanchiwai-ray) [inline .github/.jira_sync_config.yaml:21]: should we change to SOLENG-153?

Comment (Deezzir) [reply .github/.jira_sync_config.yaml:21]: Used 46 as @aieri suggested

**Review by aieri — APPROVED** [review]

**Pull request merged**